lib/server/proxyClientRequestToApi.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var Backbone = require("../application/Backbone");
+var Configure = require("../application/Configure");
+
+function proxyClientRequestToApi(apiHost){
+    return function(req,response,next){
+        if (!!req.headers['brisket-client']){
+            Configure.backbone();
+            Backbone.ajax( apiHost + req.path)
+            .then(function(data) {
```

Comment:
I wouldn't use Backbone to do this since it is server-side only and specific to express. I would likely implement this with something like [request](https://github.com/request/request).

Also, this middleware should be accompanied by unit tests.